SRC/x509parse.c

```diff
@@ -1496,8 +1497,8 @@ static void x509_hash( unsigned char *in, int len, int alg,
         case RSA_MD5   :  md5( in, len, out    ); break;
         case RSA_SHA1  : sha1( in, len, out    ); break;
         case RSA_SHA256: sha2( in, len, out, 0 ); break;
-//      case RSA_SHA384: sha2( in, len, out, 0 ); break;
-//      case RSA_SHA512: sha2( in, len, out, 0 ); break;
+        case RSA_SHA384: sha3( in, len, out, 0 ); break;
```

Comment:
if it's SHA384, the thrid parameter should be 1, I think.

Reply:
do you have what'sApp or something. Can I have your contact? Mine is +86 180 2200 9635